### DISCUSSION TITLE
How do I build an quip_params.xml parameter file?

### DISCUSSION THREAD

**Discussion by Jeffery-Li0513** [libAtoms/QUIP#465] [Q&A]
When I use QUIP, I find that core_param_file in the gap_fit program requires a file similar to quip_params.xml. But I don't know how to build this XML file. Can someone solve this problem for me?

**Comment by jameskermode** [comment]
Each potential type requires different parameters, but there are some examples [here](https://github.com/libAtoms/QUIP/blob/public/share/Parameters).
